oak-store-spi/src/main/java/org/apache/jackrabbit/oak/plugins/value/jcr/BinaryImpl.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.jackrabbit.oak.plugins.value.jcr;
 
-import static com.google.common.base.Objects.toStringHelper;
+import static org.apache.jackrabbit.guava.common.base.MoreObjects.toStringHelper;
```

Comment:
why does Objects become MoreObjects? that looks like a mistake.

Reply:
See https://guava.dev/releases/19.0/api/docs/com/google/common/base/MoreObjects.html